DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: gsDesign2
 Title: Group Sequential Design with Non-Constant Effect
-Version: 1.1.2.9
+Version: 1.1.3.0
```

Comment:
Are we planning a CRAN release soon? Otherwise I would recommend only bumping the dev version. These can be arbitrarily large numbers
```suggestion
Version: 1.1.2.10
```